### ISSUE TITLE
React Native .47 SplashScreenReactPackage.java

### ISSUE THREAD

**Issue by vivekebics** [crazycodeboy/react-native-splash-screen#82]
node_modules/react-native-splash-screen/android/src/main/java/com/cboy/rn/splashscreen/SplashScreenReactPackage.java:22: error: method does not override or implement a method from a supertype @Override

**Comment by PizzaLiu** [comment]
@vivekebics 

delete this code in SplashScreenReactPackage.java

```java


    @Override
    public List<Class<? extends JavaScriptModule>> createJSModules() {
        return Collections.emptyList();
    }

```

**Comment by devLaaziz** [comment]
+1

**Comment by Gp2mv3** [comment]
Is the Splashscreen hiding for you on Android ? I've upgraded to 0.47 and now the splashScreen never hides...

**Comment by dwilt** [comment]
@PizzaLiu your fix worked that's on, thanks. When can we expect to see this fixed and released? I am editing the files in `node_modules` and that's not a sustainable thing

**Comment by dwilt** [comment]
@crazycodeboy any update on this?

**Comment by PizzaLiu** [comment]
Hi @dwilt , you can just run `npm install react-native-splash-screen@latest --save` in the root of your project.

Or modify `"react-native-splash-screen": "^3.0.1"` in you `package.json` then `npm install`

**Comment by dwilt** [comment]
@PizzaLiu that's the version I currently have that has the problem

**Comment by dwilt** [comment]
It's [still in master of 3.0.1](https://github.com/crazycodeboy/react-native-splash-screen/blob/master/android/src/main/java/org/devio/rn/splashscreen/SplashScreenReactPackage.java#L28)

**Comment by PizzaLiu** [comment]
@dwilt  no, it is [this line](https://github.com/crazycodeboy/react-native-splash-screen/blob/f792147e61b2f43ed954f77c094a97bbfac17db6/android/src/main/java/org/devio/rn/splashscreen/SplashScreenReactPackage.java#L22) , it was fixed.

**Comment by dwilt** [comment]
@PizzaLiu got it, thanks! I was mixing this error up with [react-native-push-notification](https://github.com/zo0r/react-native-push-notification/issues/522)
